src/js/index.js

```diff
@@ -479,26 +479,35 @@ export const generate = (baseSpacing = 24, scale = 6) => {
             radius: `${baseSpacing * 0.25}px`, // 4px
           },
           pad: {
-            vertical: `${baseSpacing * 0.125}px`,
-            horizontal: `${baseSpacing * 0.5}px`,
+            vertical: `${baseSpacing * 0.125 + 1}px`, // 3px
```

Comment:
Why do we have 3px, 9px, etc values if we are following 8 pt grip system as design guidelines. Please confirm with design team once